src/draw/lv_image_decoder.h

```diff
@@ -44,6 +44,19 @@ typedef _lv_image_src_t lv_image_src_t;
 typedef uint8_t lv_image_src_t;
 #endif /*DOXYGEN*/
 
+enum _lv_image_decoder_process_flag_t {
+    LV_IMAGE_DECODER_PROCESS_FLAG_NONE = 0,
```

Comment:
As the pre/post process callbacks are application specific, I wonder if we need to add a predefined flags. 

I think having `process_state` (instead of `process flags` as `process state is more generic`) is enough.

Reply:
Done